### ISSUE TITLE
Could we add a read-only role for collaborators?

### ISSUE THREAD

**Issue by yuanhuiliu** [Apicurio/apicurio-studio#836]
Hello guys, sometimes we only want to invite someone else to review the API design without updating operations. So could we add a read-only role for collaborators? Th
![readonly](https://user-images.githubusercontent.com/16347556/60808967-1b923100-a1bc-11e9-8d1f-7434e833e09e.png)
s!

**Comment by EricWittmann** [comment]
Good idea, for sure.  I'm hoping to address a lot of this sort of thing soon by introducing projects and perhaps teams.  I know I've been saying that for awhile, but it's still a big priority and it's just a matter of finding the bandwidth.

**Comment by EricWittmann** [comment]
I'm not sure if you've noticed the new feature in Apicurio Studio that I believe may satisfy this requirement.  It's called Share Docs and is available from the action/kebab menu for every OpenAPI 2 or 3 API design:

![image](https://user-images.githubusercontent.com/1890703/68422546-69b94000-016e-11ea-87ab-99e40147b421.png)

When you click on that, you will get a dialog that will let you enable/disable sharing for the API design:

![image](https://user-images.githubusercontent.com/1890703/68422601-835a8780-016e-11ea-96bf-c569703cebb6.png)

When sharing is enabled, **any** user will be able to view the documentation for the API design.  For example, here is the link from the screenshot above:

https://studio-ws.apicur.io/sharing/76f236d3-1312-4548-a4d1-400487188c66

**Comment by yuanhuiliu** [comment]
Hello Eric, I tried this feature today, the current design can help us share the documentation, but when one documentation is shared, **anyone** can view it, which might not be secure. Besides, other team members need to visit the shared URL directly rather than forwarding to the shared page through his/her project list. What about your idea?

**Comment by EricWittmann** [comment]
OK thanks for the feedback, @yuanhuiliu - the Project & Teams update is still on the roadmap.  So hopefully we can add a read-only role when we do that work.
